store/blocks.json

```diff
@@ -0,0 +1,11 @@
+{
+  "checkout-identification": {
+    "blocks": ["image#default"]
+  },
+  "image#default": {
+    "props": {
+      "src": "https://storecomponents.vtexassets.com/arquivos/store-theme-logo-mobile.png",
```

Comment:
this is a workaround until the devtools team resolves the issues of using the `assets` builder with a "long" app name (exceeds the vbase bucket name size of 50 characters)